scripts/utils/board_v1_to_v2.py

```diff
@@ -157,11 +158,15 @@ def board_v1_to_v2(board_root, board, new_board, group, vendor, soc, variants):
 
     print(f"Creating or updating Kconfig.{new_board}...")
     new_board_kconfig_file = new_board_path / f"Kconfig.{new_board}"
-    selects = "\n\t" + "\n\t".join(["select " + setting for setting in board_soc_settings])
+    header = "# SPDX-License-Identifier: Apache-2.0\n"
+    if copyright is not None:
+        header = header + "# Copyright (c) 2024 " + copyright + "\n"
```

Comment:
hardcoding year isn't likely a good idea (although... I hope we have finished by 2025)

Reply:
Making it configurable sounded a bit over-engineered given the condition of usage.

Reply:
Resolved as I'm now retrieving previous Copyright.